docs/source/howto/aforms.rst

```diff
@@ -0,0 +1,56 @@
+.. _aforms:
+
+How to extract AcroForm fields from a PDF using Python and PDFMiner
+********************************
+
+Before you start, make sure you have :ref:`installed pdfminer.six<install>`.
+
+The second thing you need is a PDF with AcroForms (as found in PDF files with fillable forms or multiple choices).
+
+It won't work for XFA forms.
```

Comment:
Just for completeness, do you know if pdfminer.six can handle XFA forms (but in a different manner)?

I appreciate you might not have experience or interest in these forms, but we don't want to make it sound like pdfminer can't handle something if it can.

Reply:
I don't think XFA forms can be handled by pdfminer, and sadly I got no experience with them.

There's an article on SO: https://stackoverflow.com/questions/55754342/.

I understand that it's non trivial to add XFA support.